en/docs/install-and-setup/upgrading-the-solution/upgrading-wso2-api-manager-400.md

```diff
@@ -32,19 +32,22 @@
     - Go to the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources` directory.
     - Rename `wso2am-4.0.0-deployment-uk.toml` to `deployment.toml`.
     - Copy the `deployment.toml` file to the `<APIM_HOME>/repository/conf` directory and replace the existing file.
+    
+ 10. Open the `<APIM_HOME>/repository/conf/deployment.toml` file, and configure the hostnames and databases related 
```

Comment:
Let's mention to point the databases creates for Open Banking 2.0.

Reply:
Fixed in f9ac8569d3756f0ee97e73443ce1c7ea47c0aa5b